pkg/operation/seed/seed.go

```diff
@@ -37,6 +37,7 @@ import (
 	kutils "github.com/gardener/gardener/pkg/utils/kubernetes"
 	"github.com/gardener/gardener/pkg/utils/secrets"
 	utilsecrets "github.com/gardener/gardener/pkg/utils/secrets"
+	versionutils "github.com/gardener/gardener/pkg/utils/version"
```

Comment:
```suggestion
	utilversion "github.com/gardener/gardener/pkg/utils/version"
```

(analogue to `utilsecrets`, or rename this to `secretsutil`, but do it consistently everywhere)